### PR TITLE
[sdk upgrade] Upgrading to sdk usage

### PR DESCRIPTION
**Pull request by mayankamencherla**
Upgrading to sdk usage instead of using sendRequest to respective urls

https://github.com/razorpay/razorpay-magento/issues/62

### CODE REVIEW

**Comment by mayankamencherla** [comment]
@captn3m0 Done. Redirecting to failure and displaying error message.

**Comment by ccrims0n** [comment]
@mayankamencherla let's test this tomorrow.

**Comment by mayankamencherla** [comment]
@ccrims0n Cool.

File: app/code/community/Razorpay/Payments/Helper/Data.php [hunk 1]
```diff
@@ -108,24 +114,33 @@ public function createOrder($order)
             ));
         }
 
-        if (($razorpayOrderId === null) or 
-            (($razorpayOrderId) and ($this->verifyOrderAmount($razorpayOrderId, $order) === false)))
+        try
         {
-            $data = $this->getRazorpayOrderData($order);
+            if (($razorpayOrderId === null) or 
+            (($razorpayOrderId) and ($this->verifyOrderAmount($razorpayOrderId, $order) === false)))
```
Comment (ccrims0n) [inline app/code/community/Razorpay/Payments/Helper/Data.php]: alignment.

File: app/code/community/Razorpay/Payments/Helper/Data.php [hunk 2]
```diff
@@ -108,24 +114,33 @@ public function createOrder($order)
...
+        }
+        catch (Exception $e)
+        {
+            $message = 'Razorpay Error: ' . $e->getMessage();
 
-            $razorpayOrderId = $response['id'];
+            Mage::getSingleton('core/session')->addError($message);
 
-            Mage::getSingleton('core/session')->setRazorpayOrderID($razorpayOrderId);
+            $responseArray = array('error' => true);
```
Comment (ccrims0n) [inline app/code/community/Razorpay/Payments/Helper/Data.php]: @mayankamencherla return from here?

**Pull request merged**